wiki/Play_style/Tapx/id.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - tablet tap
+  - tap
+  - tekanan
+  - tablet
+  - streaming with tablet
+  - play style
+  - gaya bermain
+---
+
+# Tapx
+
+*Catatan: Gaya bermain ini membutuhkan sebuah graphic tablet.*
+
+**Tapx** adalah sebuah gaya tapping bagi pemain yang [hanya menggunakan tablet](/wiki/Play_style#tablet-only) untuk bermain pada mode [osu!](/wiki/Game_mode/osu!) dimana pemain menekan circle, meluncurkan slider, dan memutar spinner dengan menekankan pena tablet ke tablet, bersama dengan tombol keyboard/tablet kedua untuk membantu pemain dalam melakukan [streams](/wiki/Beatmap/Pattern/Stream) dan [not-not bertumpuk](/wiki/Mapping_Techniques/Stack). Metode untuk bermain ini cukuplah sulit untuk kebanyakan pemain dan tidak terlalu popular bagi kebanyakan pemain tablet.
```

Comment:
```suggestion
**Tapx** adalah sebuah gaya tapping bagi pemain yang [hanya menggunakan tablet](/wiki/Play_style#tablet-only) untuk bermain pada mode [osu!](/wiki/Game_mode/osu!) di mana pemain menekan circle, meluncurkan slider, dan memutar spinner dengan menekankan pena tablet ke tablet, bersama dengan tombol keyboard/tablet kedua untuk membantu pemain dalam melakukan [stream](/wiki/Beatmap/Pattern/Stream) dan [note-note bertumpuk](/wiki/Mapping_Techniques/Stack). Metode untuk bermain ini cukup sulit untuk kebanyakan pemain dan tidak terlalu popular bagi kebanyakan pemain tablet.
```
dimana = di mana (https://kbbi.kemdikbud.go.id/entri/di%20mana). istilah `note` itu lebih tepat tetap ditulis sebagai `note` dibandingkan dengan `not`, mirip seperti istilah yang ada di mania, di mana kita gak tulis `not panjang`, tapi `long note`